process/dataValidators/txValidator.go

```diff
@@ -88,6 +97,19 @@ func (txv *txValidator) CheckTxValidity(interceptedTx process.InterceptedTransac
 	return txv.checkAccount(interceptedTx, accountHandler)
 }
 
+func (txv *txValidator) isGuardedTxBeforeSupernova(tx data.TransactionHandler) bool {
```

Comment:
the intent/name of this method is not clear to me: 
it will return `false` for all transactions before supernova activation - a guarded transaction before supernova will return false - the name is `isGuardedTxBeforeSupernova`
but also for not guarded transactions after supernova activation.